proctor-common/src/main/java/com/indeed/proctor/common/ProctorRuleFunctions.java

```diff
@@ -80,4 +80,44 @@ public static boolean versionInRange(
         }
         return inRange(version, start, end);
     }
+
```

Comment:
Given all those code will only be used outside the proctor library. https://indeed-pte.slack.com/archives/C1F00STA9/p1688590802320449?thread_ts=1688582115.448299&cid=C1F00STA9

Can we add java doc to each of those methods to explain it is not meant to be called inside proctor lib?

Reply:
actually, can we create a subclass and only the subclass has those extra methods?